packages/components/details/template.njk

```diff
@@ -5,11 +5,14 @@
 {{- nhsukAttributes(params.attributes) }} {{- " open" if params.open }}>
   <summary class="nhsuk-details__summary">
     <span class="nhsuk-details__summary-text">
-      {{ params.text }}
+      {{ params.summaryHtml | safe | trim | indent(6) if params.summaryHtml else params.summaryText }}
```

Comment:
Is it worth considering a nest object here, eg

```njk
{{ params.summary.html | safe | trim | indent(6) if params.summary.html else params.summary.text }}
```

Main advantage would be forward-compatibility with any other params added to the summary, eg `classes` or `attributes`? Plus it’s more consistent with the general pattern.

Downside is a deviation from the govuk component.

Reply:
I think probably better to align with GOV.UK?